NOTICE.txt

```diff
@@ -41,13 +41,9 @@ Apache-2.0
   accessors-smart-2.5.0.jar
   annotations-17.0.0.jar
   arrow-format-13.0.0.jar
-  arrow-format-15.0.0.jar
   arrow-memory-core-13.0.0.jar
-  arrow-memory-core-15.0.0.jar
   arrow-memory-netty-13.0.0.jar
-  arrow-memory-netty-15.0.0.jar
   arrow-vector-13.0.0.jar
-  arrow-vector-15.0.0.jar
```

Comment:
does this mean that before we were shipping both 13 and 15 arrow?

Reply:
I was also confused about this, but I assume so yes.

Reply:
but that would be a problem and a source of ooms in that case (if we would assume that its not plain arrow by default doing really strange things). Also in that preview i can see that we have the duplication of other libs as well

Reply:
Fixed here https://github.com/neo4j/apoc/pull/593